crates/ruff/src/rule_selector.rs

```diff
@@ -184,6 +184,20 @@ impl JsonSchema for RuleSelector {
                 std::iter::once("ALL".to_string())
                     .chain(
                         RuleCodePrefix::iter()
+                            .filter(|p| {
```

Comment:
Ahh I see, so we want to omit these because as-is, they're _included_ when running the tests with `--all-features`, but _excluded_ when running `cargo dev generate-all` (and, intentionally, excluded from the docs, schema, etc.).